app/pages/home-not-logged-in/discover.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import counterpart from 'counterpart';
+
+counterpart.registerTranslations('en', {
+  discoverHomePage: {
+    appAnnounce: 'Bring the Zooniverse with you. Download the app for iOS and Android devices.',
+    discover: 'Discover, teach, and learn',
+    doScience: 'Do science, anywhere',
+    signIn: 'Sign in',
+    register: 'Register',
+    research: `
+      The Zooniverse enables everyone to take part in real cutting edge
+      research in many fields across the sciences, humanities, and more.
+      The Zooniverse creates opportunities for you to unlock answers and
+      contribute to real discoveries.
+    `,
+    what: 'What\'s this?'
+  }
+});
+
+const HomePageDiscover = ({ showDialog }) => {
+  return (
+    <section className="home-discover">
+      <div className="home-discover__content">
+        <Translate className="secondary-kicker" content="discoverHomePage.what" />
+        <br />
+        <Translate className="tertiary-headline" content="discoverHomePage.discover" />
+
+        <Translate className="display-body" component="p" content="discoverHomePage.research" />
+
+        <button type="button" value="sign-in" className="primary-button" onClick={showDialog}>
+          <Translate content="discoverHomePage.signIn" />
+        </button>
+
+        <button type="button" value="register" className="primary-button primary-button--light" onClick={showDialog}>
+          <Translate content="discoverHomePage.register" />
+        </button>
+
+        <hr />
+
+        <Translate className="tertiary-kicker" content="discoverHomePage.doScience" />
```

Comment:
Yes, this is the heading for a subsection, so I can attach a header component to this element.